day-1/files/staging/hetzner-staging-alpha-01/hoprd.json

```diff
@@ -0,0 +1,11 @@
+{
+  ".hoprd.id": {
+    "peer_id": "12D3KooWNe4N2isa7V4T9YhBxuPCb8jAdQWMQXSHRsHAXBKgAENs",
+    "packet_key": "0xbe83fd433f255f7750852d4b61321a32acb43f93de24cd282f11d3df566ac860",
+    "chain_key": "0x031d1c0d9d79802f11b844765f39fd9362293991eadb783c210f415e86ae78b155",
```

Comment:
No, those values are not sensitive. The file is not even copied to the servers. I added here as a reference, because we can get from there the `native_address`